src/instrumentation.ts

```diff
@@ -69,7 +76,7 @@ export const logDbError = ({ logger }: ParsedSettings, e: Error): void => {
       chalk.red.bold(`🛑 Error occurred whilst processing migration`),
     );
   }
-  const { severity, code, detail, hint } = e as any;
+  const { severity, code, detail, hint } = e as InstrumentationError;
```

Comment:
Change the function declaration to:

```
export const logDbError = ({ logger }: ParsedSettings, error: Error): void => {
  const e = error as InstrumentationError;
  e._gmlogged = true;
```

Then we can:

```suggestion
  const { severity, code, detail, hint } = e;
```